db/migrate/20210722111142_remove_rhsm_columns.rb

```diff
@@ -0,0 +1,41 @@
+class RemoveRhsmColumns < ActiveRecord::Migration[6.0]
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    change_table "miq_databases" do |t|
+      t.remove "registration_type"
+      t.remove "registration_organization"
+      t.remove "registration_server"
+      t.remove "registration_http_proxy_server"
+    end
+
+    Authentication.where(:authtype => %w[registration registration_http_proxy])
+                  .destroy_all
+
+    change_table "authentications" do |t|
+      t.remove "rhsm_sku"
+      t.remove "rhsm_pool_id"
+      t.remove "rhsm_server"
```

Comment:
These are unrelated.  They were added in https://github.com/ManageIQ/manageiq/pull/7779/files#diff-49eb1e9f4c7c64a5250f45cd08b58cd7e0832c08f18c26e06082a972b8d183e7R29-R31

Reply:
Although it looks like the feature requiring them was removed in https://github.com/ManageIQ/manageiq/pull/18515/files#diff-031eafbc5af9f5092abc3e9c07627e482c156a37c0c9e5ff798089e0e61b0464L1

Reply:
Oh that's right - this is related to ovirt authetnications. @agrare can you chime in here?

Reply:
I think that's actually code for deploying openshift via automate (that's pretty cool actually)

Wasn't aware that existed before, but no it is definitely no longer used

Reply:
If that's the case can _all_ of these go away? https://github.com/ManageIQ/manageiq/pull/7779/files#diff-49eb1e9f4c7c64a5250f45cd08b58cd7e0832c08f18c26e06082a972b8d183e7R6-R31

Reply:
Maybe?  I didn't check that none of those are used anymore.

Either way probably should be pulled out of this PR since it isn't related to rhn/etc...

Reply:
> Maybe? I didn't check that none of those are used anymore.

They all went in at the same time for the same feature, so if the feature is gone, I assumed they all should go

> Either way probably should be pulled out of this PR since it isn't related to rhn/etc...

Totally agree - doesn't belong in this PR.

Reply:
Can't remember who asked me to delete `RegistrationSystem`  "while I was at it"

So the goal is now to separate out the `UpdateSystem` from the `RegistrationSystem` and actually grow the `RegistrationSystem` (satellite) PR to delete more fields?

Reply:
There are three completely different "satellite/registration" things in our product that are unrelated to each other.

1. Our appliance had the ability to update the RPMs on the appliance itself
2. The Satellite/Foreman provider is used for managing satellites
3. The OpenShift provider had a capability to be able to deploy new openshifts, and part of that effort was the need to store various credentials for that purpose (including credentials to an rhsm system)

This PR should focus only on number 1. The `rhsm_` columns in the authentications table are part of number 3.  I can't recall offhand which number `RegistrationSystem` belongs to, but I _thought_ it was number 1 ...perhaps @bdunne knows better.

Reply:
Yeah, `RegistrationSystem` is part of 1